### PR TITLE
Move carta startup script outside image.

### PR DESCRIPTION
**Pull request by at88mph**

### CODE REVIEW

File: deployment/k8s-config/kustomize/base/skaha/config/launch-carta.yaml [hunk 1]
```diff
@@ -72,6 +72,8 @@ spec:
           protocol: TCP
           name: frontend-port
         volumeMounts:
+        - mountPath: "/skaha-system"
+          name: start-desktop
```
Comment (brianmajor) [inline deployment/k8s-config/kustomize/base/skaha/config/launch-carta.yaml]: typo... 'start-carta'... and in the volumes below

**Review by brianmajor — CHANGES_REQUESTED** [review]

File: deployment/k8s-config/kustomize/base/skaha/config/launch-carta.yaml (hunk 1 above)
Comment (at88mph) [reply deployment/k8s-config/kustomize/base/skaha/config/launch-carta.yaml]: I just kept it consistent with the other launch scripts.  I will change.

File: deployment/k8s-config/kustomize/base/skaha/config/launch-carta.yaml [hunk 2]
```diff
@@ -95,6 +97,10 @@ spec:
           readOnly: false          
       - name: scratch-dir
         emptyDir: {}
+      - name: start-carta
+        configMap:
+          name: start-desktop
```
Comment (brianmajor) [inline deployment/k8s-config/kustomize/base/skaha/config/launch-carta.yaml]: This probably needs to be changed to start-carta too?
I think the script is called `skaha-carta.sh`, but not sure about the configmap name as helm creates it.

**Review by brianmajor — CHANGES_REQUESTED** [review]

File: deployment/k8s-config/kustomize/base/skaha/config/launch-carta.yaml (hunk 2 above)
Comment (at88mph) [reply deployment/k8s-config/kustomize/base/skaha/config/launch-carta.yaml]: I've created a new config map for `carta` now.

**Review by brianmajor — APPROVED** [review]

**Pull request merged**